crypto/proof_test.go

```diff
@@ -29,6 +30,26 @@ func TestPath(t *testing.T) {
 	}
 }
 
+// can we also store sth. else in the leaves (not only hashIds)? The tree
```

Comment:
`HashID` is only a type alias to a slice of bytes so to me a slice of bytes is pretty agnostic already ;) Most of the time you don't even need to cast to `HashID` golang does it for you.